src/main/kotlin/it/pagopa/wallet/documents/migration/WalletPaymentManagerDocument.kt

```diff
@@ -0,0 +1,14 @@
+package it.pagopa.wallet.documents.migration
+
+import java.time.Instant
+import org.springframework.data.annotation.CreatedDate
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
+
+@Document("wallets-migration-pm")
+data class WalletPaymentManagerDocument(
+    @Id var walletPmId: String,
+    var walletId: String,
+    var contractId: String,
+    @CreatedDate var createdAt: Instant,
```

Comment:
To keep the same nomenclature as the other documents for example **walletDocument**, wouldn't it be better to call it creationDate ?
```suggestion
    @CreatedDate var creationDate: Instant,
```